modules/wallet/encrypt.go

```diff
@@ -120,14 +130,30 @@ func (w *Wallet) unlock(masterKey crypto.TwofishKey) error {
 	return nil
 }
 
+// wipeSecrets erases all of the seeds and secret keys in the wallet.
+func (w *Wallet) wipeSecrets() {
+	// 'for i := range' must be used to prevent copies of secret data from
+	// being made.
```

Comment:
since you're only accessing the `SecretKeys` field, which is a pointer, a copy should okay. nbd